scopes/preview/preview/generate-link.ts

```diff
@@ -35,7 +35,7 @@ export function generateLink(
   }
 
   return `
-import { linkModules } from '${toWindowsCompatiblePath(require.resolve('./preview.preview.runtime'))}';
+import { linkModules } from '@teambit/preview/dist/preview.preview.runtime';
```

Comment:
I'm going to make this a new component soon.
Extracting the link modules to its own component. Then we should update this line